.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release
+
+on:
+    push:
+        branches: [main]
+
+jobs:
+    release:
+        name: Release
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+            - name: Setup Node.js
+              uses: actions/setup-node@v1
+              with:
+                  node-version: 14.x
+            - name: Install dependencies
+              run: npm install
```

Comment:
Similar tasks as in [#10](https://github.com/Safe-Security/unleash-client/pull/10) in unleash-client repo. Moving to `yarn`

Reply:
Done